### ISSUE TITLE
the yolov3 train weights cannot work on another computer

### ISSUE THREAD

**Issue by xiaose26** [pjreddie/darknet#1330]
I get the weights by training yolov3 on my own database,while I test the weights,on it's trained computer,work correct even perfect,but ,I test the same weight on another computer,there isn't any box on the video,why?
the environment is opencv3.4,cuda9.0,python3.6 and GTX1080,the yolov3 network is origin author,not AB

**Comment by theshawnz** [comment]
Have you ruled out that the issue may be configuration of your camera? I found that my cameras settings matter. How about creating a video on your test machine (from the camera that gives you video with detected objects) and testing that video (mp4) on the new machine? this way you could rule out that the issue is not related to the configuration of your video input.

**Comment by xiaose26** [comment]
yeah,I test the weights on another computer by video(mp4),no object detected. then I shutdown the gpu or cudnn,nothing changed.now,I  want to train the yolov3 again on another computer.

**Comment by xiaose26** [comment]
okay，I have solved this problem,you must put the gcc and g++ version below 6.0,and the nvidia architecture must adapt to your GPU(my gpu is GTX1080ti,so cuda_arch_bin="6.1" and compute be set 61).now ,it works.thanks!!
